Changes Text to be more simple and Website now Responsive

maguzzz opened the pull request:
* Changed the  `project description` to be more comprehensive 
* Changed About me text to be more readable on phones
* Fixed the website  `Responsivess`

Boby900 commented on index.css:
```diff
@@ -192,9 +189,19 @@ h2{
+    .project{
+        margin-bottom: 8.5em;
+    }
+    .info{
```
thanks @maguzzz ,
 but  this line of code is disrupting the text alignment,
![Screenshot (27)](https://user-images.githubusercontent.com/111265239/212581033-5cdf2f07-348f-42a5-b44f-35657b70b3e1.png)

maguzzz commented:
Ah sry looks like i changed the about me text after i fixed the responsiveness of the projects and forgot to check if its still works.

I will fix it today!!!

Boby900 commented:
thanks once again @maguzzz
